torch/testing/_internal/common_methods_invocations.py

```diff
@@ -21027,6 +21034,12 @@ def __init__(
         "_refs.where",
         torch_opinfo_name="where",
         op=lambda self, condition, other: refs.where(condition, self, other),
+        skips=(
```

Comment:
supports_nvfuser=False should be a more precise skip than skipping the entire executor test